manifests/cron.pp

```diff
@@ -22,10 +26,13 @@
     $scout_roles_switch = " -r ${roles}"
   }
 
+  $command = "/usr/bin/env scout \${SCOUT_KEY}${scout_environment_switch}${scout_roles_switch}"
```

Comment:
Are we able to simplify this with `join` and strip out the whitespace from the other variables so they aren't all bunched up together?

```puppet
join([$SCOUT_KEY, $scout_environment_switch, $scout_roles_switch], ' ')